code/modules/mob/new_player/new_player.dm

```diff
@@ -355,8 +355,13 @@
 
 	//Find our spawning point.
 	var/list/join_props = job_master.LateSpawn(client, rank)
+	
+	if(!join_props)
+		return
+
 	var/turf/T = join_props["turf"]
 	var/join_message = join_props["msg"]
+	var/announce_channel = join_props["channel"] || "Common"

```

Comment:
Is there something in here that's supposed to be using this? Because every time the code compiles it's throwing a warning that this variable isn't used.

Reply:
It's probably a thing used downstream that got left in here.
Might be a good idea to make Travis fail on warnings as well somehow.